requirements.txt

```diff
@@ -0,0 +1,7 @@
+This file describe the pypi packages needed to execute these examples.
```

Comment:
this should be pip freeze content.

eg
pyats == 4.0.0
etc

you can add comments, though.